framework/sandstone.cpp

```diff
@@ -2352,7 +2352,7 @@ static void list_tests(int opt)
     auto groups = collate_test_groups();
     int i = 0;
 
-    for (struct test *test = test_set.begin(); test != test_set.end(); ++test) {
+    for (auto test = test_set.begin(); test != test_set.end(); ++test) {
```

Comment:
This could also be changed to range based for loop.

Reply:
Indeed, but I went for the minimal change here. Changing to a for-range loop would change the body of the for to use `test.` instead of `test->`